core/server/src/test/java/alluxio/AbstractClientTest.java

```diff
@@ -57,14 +64,12 @@ public void checkVersion(AlluxioService.Client thriftClient, long version) throw
   public void unsupportedVersion() throws Exception {
     final AlluxioService.Client thriftClient = Mockito.mock(AlluxioService.Client.class);
     Mockito.when(thriftClient.getServiceVersion()).thenReturn(1L);
-
-    try (TestClient client = new TestClient()) {
-      client.checkVersion(thriftClient, 0);
-      Assert.fail("checkVersion() should fail");
-    } catch (IOException e) {
-      Assert.assertEquals(ExceptionMessage.INCOMPATIBLE_VERSION.getMessage(SERVICE_NAME, 0, 1),
-          e.getMessage());
-    }
+    mExpectedException.expect(IOException.class);
+    mExpectedException.expectMessage(ExceptionMessage.INCOMPATIBLE_VERSION.getMessage(SERVICE_NAME,
+        0, 1));
```

Comment:
Could you update the formatting here? I think we would want to have a new line after `expectMessage(`